src/OSPSuite.Core/Domain/Populations/Covariates.cs

```diff
@@ -0,0 +1,22 @@
+﻿using OSPSuite.Utility.Collections;
+
+namespace OSPSuite.Core.Domain.Populations
+{
+   public class Covariates
```

Comment:
We will have to do some mapping in PKSim as the covariates object used Gender and Population object that are filled directly from the database